modules/rostests/winetests/comctl32/static.c

```diff
@@ -20,7 +20,9 @@
 #include <stdarg.h>
 #include <stdio.h>
 
+#ifndef __REACTOS__
 #define STRICT
+#endif
```

Comment:
NOTE: This restores the guards that were removed in the sync commit 0707475f6974a0add99fc396146c26a970c5148b